app/assets/javascripts/discourse/ember/resolver.js.es6

```diff
@@ -188,7 +188,8 @@ export default Ember.DefaultResolver.extend({
       const dashed = decamelized.replace(/_/g, '-');
       return Ember.TEMPLATES[decamelized] ||
              Ember.TEMPLATES[dashed] ||
-             Ember.TEMPLATES[dashed.replace('admin-', 'admin/')];
+             Ember.TEMPLATES[dashed.replace('admin-', 'admin/')] ||
+             Ember.TEMPLATES[dashed.replace('admin-', 'admin/templates/')];
```

Comment:
Oh, maybe of note: The way I'm adding that admin route is to put these lines at the end of the `admin-route-map.js.es6` `map()` function, which seemed to be consistent with the rest of the file:

```
    this.resource('adminChats', { path: '/chats' }, function() {
      this.resource('adminChat', { path: '/:id' })
    });
```